go/vt/sqlannotations/sqlannotations.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2015, Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package sqlannotations provides functions
+// for annotating DML statements with keyspace-id
+// comments and parsing them. These annotations
+// are used during filtered-replication to route
+// the DML statement to the correct shard.
+package sqlannotations
```

Comment:
I don't think it's 100% necessary, but maybe unit-testing this library locally would be a bit better. I know coverage is probably good enough for positive cases with the other tests you added, but error cases are probably not covered.